content/components/page-header.mdx

```diff
@@ -0,0 +1,330 @@
+---
+title: Page header
+reactId: drafts_page_header
+description: Page header determines the top-level headings of a UI
+figma: https://www.figma.com/file/Ee0OrXuOLXMDqUW83EnDHP/PageHeader-(FY23-Q1)?node-id=0%3A1&t=8cKRbkNYhwbqEoHj-1
+---
+
+import {Box, Text} from '@primer/react'
+import ComponentLayout from '~/src/layouts/component-layout'
+export default ComponentLayout
+
+## Usage
+
+Page header is used to determine the arrangement of top-level headings and to determine how elements adopt to different viewports. Page header organizes different elements that apply context and provide navigation for the current UI. 
+
+The elements within Page header are rearranged and displayed accordingly depending on the [viewport size](/foundations/layout#viewport-ranges) and context of the page.
+
+<img 
+    src="https://user-images.githubusercontent.com/586552/222194474-74b72ae3-f8d4-4336-9a2c-9d5c18d91c42.png"
+    role="presentation"
+    width="960"
+    alt="Page header component"
+/>
+
+## Anatomy 
+
+Page header can include a Context bar, Title bar, description, and contextual navigation (pertaining to the current page). 
+
+By default, the [Context bar](#context-bar) is visible on narrow viewports only, the [Title bar](#title-bar) is visible on all viewports, and the [Back button and Trailing action](#title-bar-action-regions) are visible on regular viewports.
+
+<img
+    src="https://user-images.githubusercontent.com/586552/221939296-410d3b17-ccbb-48ae-b1cf-c3a8cdcfcaa5.png"
+    role="presentation"
+    width="993"
+    alt="Page header anatomy"
+/>
+
+## Context bar
+
+The Context bar is intended to set context for the user on smaller viewports and provide any actions that fit within the Context bar. As a reminder, by default, the Context bar (including its actions) is only visible on narrow viewports. 
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Parent link
+    </Text>
+    <Text as="p" mt="0">
+        The default way to let users navigate up in the hierarchy on narrow viewports.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/586552/222194471-dfc56e9a-a2ab-48a0-8a04-41ebb5ec4adf.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Context bar actions
+    </Text>
+    <Text as="p" mt="1">
+        Actions that appear in the Context bar specifically.
```

Comment:
I think the only reason this distinction (taken from Figma) is because they are actions that live specifically in the context bar, vs the actions in the title bar (which appear below). I don't believe there are any differences between the actions here vs the title bar - except that they pertain to their context

Reply:
If the context bar is only visible on narrow viewports, what do we do with those actions for wider viewports? That might be good info for here.

Neither of us was very involved in the design of this component, so we might have to just merge and leave things vague until we get some answers.